module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderLink.kt

```diff
@@ -0,0 +1,21 @@
+package org.cru.godtools.shared.renderer.content
+
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import org.cru.godtools.shared.tool.parser.model.Link
+
+@Composable
+internal fun ColumnScope.RenderLink(link: Link) = RenderLinkNode(
+    link,
+    Modifier
+        .padding(horizontal = Horizontal_Padding)
+        .fillMaxWidth(),
+)
+
+@Composable
+internal fun RenderLinkNode(link: Link, modifier: Modifier = Modifier) {
```

Comment:
you shouldn't need to use a separate `RenderLinkNode` composable here. I did it for Text since Text is used so many places